scripts/packaging/generate_asset_cache.py

```diff
@@ -17,11 +17,19 @@
 """Generate and package the asset cache"""
 import os
 from pathlib import Path
-import re
 import sys
 import shutil
 import hashlib
 
+EXTENSIONS = [
+    'jpg', 'JPG', 'png', 'PNG', 'jpeg', 'JPEG', 'stl', 'STL', 'dae', 'DAE', 'obj', 'OBJ', 'mp3', 'MP3', 'wav', 'WAV',
+    'hdr', 'HDR', 'fbx', 'FBX'
```

Comment:
I believe we can remove all the uppercase versions as we don't have and won't have uppercase extensions in the Webots distribution. Also, I believe we can remove the `.jpeg` which we also don't and won't have.